src/hooks/useSettings.ts

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useMemo, useState } from "react";
+
+import { DEFAULT_SHOW_CHAT } from "settings";
+
+import { Settings } from "types/Firestore";
+import { SparkleSelector } from "types/SparkleSelector";
+
+import { isTruthy } from "utils/types";
+
+import { useFirestoreConnect } from "hooks/useFirestoreConnect";
```

Comment:
```suggestion
import { useFirestoreConnect, isLoaded } from "hooks/useFirestoreConnect";
```

Reply:
the commit is here: https://github.com/sparkletown/sparkle/pull/2261/commits/e37f952af6f7c168cafd88fd5a8710c09d0bcfd6

Thank youuuuuu!! ❤️